### PR TITLE
Fixes #10 - Make code compatible with Python 3

### LINKED ISSUE #10: Add Python 3 support
This project is to be used by other projects, and for maximum compatibility Python 3 support should be added. One example of such project would be [autowebcompat](https://github.com/marco-c/autowebcompat), which works with Python 3.

### PR DESCRIPTION
**Pull request by skulltech**
Attempts to resolve #10 

So far took care of only `extract_id_title_url.py`. More commits to come.

### CODE REVIEW

File: extract_id_title_url.py [hunk 1]
```diff
@@ -19,7 +19,10 @@
 import re
 import socket
 import sys
-import urllib2
+try:
+    from urllib.request import urlopen, Request
+except ImportError:
+    from urllib2 import urlopen, Request
```
Comment (karlcow) [inline extract_id_title_url.py]: Could you put import on separate lines for each module.

**Review by karlcow — CHANGES_REQUESTED** [review]
@SkullTech Thanks for this. A simple modification, more stylistic than everything else.

File: extract_id_title_url.py (hunk 1 above)
Comment (skulltech) [reply extract_id_title_url.py]: I didn't get what you meant. :/

**Comment by skulltech** [comment]
Also, the `dump_webcompat_to_db` script uses a module named `db`.

```python
from db import db_session
import json

DB_PATH = 'sqlite:////tmp/test.db'
Base = declarative_base()
Base.query = db_session.query_property()
```

The `db` module is pip3 installable, but it is not compatible with Python 3. Also, I couldn't find the module (it's documentation or source code). So to make the script usable, we'll have to use something else for Database access. [Peewee](https://peewee.readthedocs.io/en/latest/) would be a good option, as it's simple but effective.

File: extract_id_title_url.py (hunk 1 above)
Comment (karlcow) [reply extract_id_title_url.py]:
```python
try:
    from urllib.request import urlopen
    from urllib.request import Request
except ImportError:
    from urllib2 import urlopen
    from urllib2 import Request
```

Comment (karlcow) [reply extract_id_title_url.py]: ooops I even meant

```python
try:
    from urllib.request import Request
    from urllib.request import urlopen
except ImportError:
    from urllib2 import Request
    from urllib2 import urlopen
```

* one line by import
* alphabetical order

Comment (skulltech) [reply extract_id_title_url.py]: Oh, I got it. :D Done!

**Comment by skulltech** [comment]
For now, we can skip `dump_webcompat_to_db.py` script as it cannot be ported to Python 3 as it is.

**Comment by karlcow** [comment]
Note that you do not need db module at all. 
see how we do it with only SQLAlchemy in webcompat.com

https://github.com/webcompat/webcompat.com/blob/4e9568c3591626007b369a64bbea4d680eea7d1a/webcompat/db/__init__.py#L29-L34

**Comment by skulltech** [comment]
@karlcow I've made changes as you requested. Also fixed the `dump_webcompat_to_db.py` script following [the file you referenced](https://github.com/webcompat/webcompat.com/blob/4e9568c3591626007b369a64bbea4d680eea7d1a/webcompat/db/__init__.py). Please review the changes.

**Review by karlcow — APPROVED** [review]
Thanks a lot! Good work.

**Pull request merged**